.github/sync-param-files.yaml

```
@@ -28,3 +28,8 @@
       dest: system_launch/config/system_monitor/ntp_monitor.param.yaml
     - source: system/system_monitor/config/process_monitor.param.yaml
       dest: system_launch/config/system_monitor/process_monitor.param.yaml
+
+- repository: autowarefoundation/autoware
```

By the way, is here wrong?

```suggestion
- repository: autowarefoundation/autoware_launch
```

---

Yes it seems wrong